### PR TITLE
CI tasks cleanup

### PR DESCRIPTION
**Pull request by martinpitt**
With this we can re-use the same make rules for the rhel-8 and other branches, and they become extensible for other containers.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
I don't really understand the first commit, but the second definitely LGTM.

File: Makefile.am [hunk 1]
```diff
@@ -80,14 +80,15 @@ RPM_BUILD_DIR = $(BUILD_RESULT_DIR)/01-rpm-build
...
+CONTAINER_ENGINE ?= podman
 CONTAINER_BUILD_ARGS ?=
 CONTAINER_TEST_ARGS ?=
 
 # ci-tasks container
 CI_TASKS_DOCKERFILE ?= $(srcdir)/dockerfile/ci-tasks
 CI_TASKS_NAME ?= anaconda/ci-tasks
 CI_TASKS_TAG ?= $(GIT_BRANCH)
-CI_TASKS_TEST_ARGS ?= --rm -v .:/anaconda:Z
+CI_TASKS_TEST_ARGS ?= --rm -v $(CURDIR):/anaconda:Z
```
Comment (jkonecny12) [inline Makefile.am:91]: I did not know about this variable. Pretty nice!

Don't we want to use this instead of ``$(srcdir)`` above too?

**Review by jkonecny12 — APPROVED** [review]
Looks good to me otherwise.

File: Makefile.am (hunk 1 above)
Comment (martinpitt) [reply Makefile.am]: Good point, thanks! I cleaned this up by having srcdir default to `$(CURDIR)` instead of `.` and using `$(srcdir)` here.

**Review by jkonecny12 — APPROVED** [review]
Looks great now. Thanks!

**Pull request merged**